app/interactors/import/create_batch.rb

```diff
@@ -1,40 +1,37 @@
-class Import::CreateBatch
-  include Interactor
+require 'creator'
 
-  before :create_batch
-  before :validate
+class Import::CreateBatch
+  include Interactor::Creator
 
   def call
-    # do save in before block
+    model.whodunnit(context.user) { model.save! }
 
-    # queue file processing
+    # TODO: Queue processing
```

Comment:
Is this TODO needed for the master branch?
Might be better to make it an issue in Github if it is going to be a new feature.